recipes/fibertaskinglib/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class FiberTaskingLibConan(ConanFile):
+    name = "fibertaskinglib"
+    description = "enabling task-based multi-threading. It allows execution of task graphs with arbitrary dependencies."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/RichieSams/FiberTaskingLib"
+    topics = ("coroutine", "fiber", "task-scheduler")
+    package_type = "static-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_cpp17": [True, False],
+    }
+    default_options = {
+        "with_cpp17": False,
```

Comment:
I see why you didn't rely in the cppstd from the compiler, but I fear this might be an antipattern, let me quickly check with the team :)